docs/eventing/experimental-features.md

```diff
@@ -51,4 +51,30 @@ data:
 
 ## Features list
 
-<!-- TODO there are no experimental features at the moment -->
+### `KReference.Group` field
+
+**Flag name**: `kreference-group`
+
+**State**: Alpha, disabled by default
+
+When using the `KReference` type to refer to another Knative resource, you can just specify the API `Group` of the resource, instead of the full `APIVersion`.
+
+For example, in order to refer to an `InMemoryChannel`, more than using:
+
+```yaml
+apiVersion: messaging.knative.dev/v1
+kind: InMemoryChannel
+name: my-channel
+```
+
+You can use instead:
+
+```yaml
+group: messaging.knative.dev
+kind: InMemoryChannel
+name: my-channel
+```
+
+Thanks to this feature you can let Knative resolve the full `APIVersion` and further upgrades/deprecations/removals of the referred CRD won't affect your resources.
+
+Note: At the moment this feature is implemented only for `Subscription.Spec.Subscriber.Ref`, in future other usages of `KReference` will support it.
```

Comment:
```suggestion
!!! note
    At the moment this feature is implemented only for `Subscription.Spec.Subscriber.Ref`.
```

We tend not to mention future plans as best practice in docs

Reply:
There is a plan to follow for this feature which includes using that field pretty much everywhere in our apis... I do wonder, maybe should I link the issue somewhere here in the docs?

Reply:
@slinkydeveloper please open docs issues for any future additions so we can track it and make sure it gets added rather than mentioning it here 🙂

Reply:
I meant adding a link here in this paragraph, let me add it so you see what i mean

Reply:
Oh, I see. No, please don't link to upcoming issues in the docs, it's the same really as mentioning future plans and we should avoid that.

Reply:
Check it out now